### DISCUSSION TITLE
Upgrade to Material UI 5.0

### DISCUSSION THREAD

**Discussion by juanpicado** [verdaccio/verdaccio#2448] [Ideas]
https://mui.com/blog/mui-core-v5/

PR welcome 🙃

**Comment by sergiohgz** [comment]
We should talk about some changes done in MUI5 that could benefit UI customization

**Reply by juanpicado** [reply]
A shallow evaluation should be useful, I haven't went through the article but maybe the hardest part are the grid and suggestions.
